cmd/offboard/output.go

```diff
@@ -0,0 +1,87 @@
+package offboard
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/open-sauced/pizza-cli/v2/pkg/config"
+	"github.com/open-sauced/pizza-cli/v2/pkg/utils"
+)
+
+func generateConfigFile(outputPath string, attributionMap map[string][]string) error {
+	file, err := os.Create(outputPath)
+	if err != nil {
+		return fmt.Errorf("error creating %s file: %w", outputPath, err)
+	}
+	defer file.Close()
+
+	var config config.Spec
+	config.Attributions = attributionMap
+
+	// for pretty print test
+	yaml, err := utils.OutputYAML(config)
+
+	if err != nil {
+		return fmt.Errorf("failed to turn into YAML: %w", err)
+	}
+
+	_, err = file.WriteString(yaml)
+
+	if err != nil {
+		return fmt.Errorf("failed to turn into YAML: %w", err)
+	}
+
+	return nil
+}
+
+func generateOwnersFile(path string, offboardingUsers []string) error {
```

Comment:
nit: This functionality is also duplicated from `generateOutputFile` in `cmd/generate/codeowners/output.go`. It'd make sense to hoist this up also in the future.